interfaces/builtin/utils.go

```diff
@@ -91,3 +91,39 @@ func udevUsbDeviceSnippet(subsystem string, usbVendor int64, usbProduct int64, k
 func udevSnapSecurityName(snapName string, appName string) string {
 	return fmt.Sprintf(`snap_%s_%s`, snapName, appName)
 }
+
+func ensureSlotIfaceMatch(iface interfaces.Interface, slot *interfaces.Slot) {
```

Comment:
They are tested indirectly but I'll add a small set of tests. Thanks for spotting that.